reactome_base_sonatype/build.xml

```diff
@@ -0,0 +1,200 @@
+<!-- based on instructions from https://central.sonatype.org/pages/apache-ant.html -->
+<project name="reactome-base" default="deploy" basedir="." xmlns:artifact="antlib:org.apache.maven.artifact.ant">
+	<property name="projectDir" value=".." />
+	<property name="srcDir" value="${projectDir}/src" />
+	<property name="libDir" value="${basedir}/lib" />
+	<property name="classDir" value="${basedir}/classes" />
+	<property name="dist" location="dist" />
+
+	<!-- define Maven coordinates -->
+	<property name="groupId" value="org.reactome.base" />
+	<property name="artifactId" value="reactome-base" />
+	<property name="version" value="1.2.22-SNAPSHOT" />
```

Comment:
Should the snapshot naming convention be used?

Reply:
Yes for this particular version because the first deployment to Sonatype was to the snapshot repository.  It will need to be changed if running the target to deploy to Sonatype as a release.